spa/src/common/LoggedinInfo.tsx

```diff
@@ -22,10 +22,16 @@ const LoggedinInfo = ({ username, logout }: { username: string; logout: () => vo
 		analyticsClient.sendUIEvent({ actionSubject: "switchGitHubAccount", action: "clicked" });
 	};
 
-	return <LoggedInContent>
-		<div data-testid="logged-in-as">Logged in as <b>{username}</b>.&nbsp;</div>
-		<Button style={{ paddingLeft: 0 }} appearance="link" onClick={clicked}>Change GitHub login</Button>
-	</LoggedInContent>;
+	return (
+		<div css={loggedInContentStyle}>
```

Comment:
This div doesn't pick up the css styling, missing the header import `/** @jsxImportSource @emotion/react */`

Reply:
Somehow it gt removed